pkg/nodebootstrap/assets/bootstrap.ubuntu.sh

```diff
@@ -18,6 +35,10 @@ systemctl reset-failed
   source /etc/eksctl/kubelet.env
   source /etc/eksctl/metadata.env
 
+  if [[ -z "${MAX_PODS+x}" ]];
```

Comment:
Let's move it below as `"max-pods=${MAX_PODS:-$(get_max_pods ${INSTANCE_TYPE})}"`